docs/src/osdk/reference/commands/profile.md

```diff
@@ -0,0 +1,69 @@
+# cargo osdk profile
+
+## Overview
+
+The profile command is used to collect stack traces when running the target
+kernel in QEMU. It attaches to the GDB server initiated with the run subcommand
+and collects the stack trace periodically. The collected data can be
+further analyzed using tools like
+[flame graph](https://github.com/brendangregg/FlameGraph).
```

Comment:
You think it makes sense to install this tool in our Docker image?

Reply:
I found a great Rust crate [inferno](https://crates.io/crates/inferno), so no need to use the perl one.